sites/mosul/configs/openmrs/initializer_config/ampathforms/F11-Family_Planning_Assessment.json

```diff
@@ -454,79 +455,79 @@
               "required": false,
               "questionOptions": {
                 "rendering": "multiCheckbox",
-                "concept": "conditions",
+                "concept": "9deb65e1-8709-41a5-92a5-f48e18237516",
                 "answers": [
                   {
-                    "label": "Latex allergy (patient or partner)",
-                    "concept": "latexAllergy"
-                  },
-                  {
-                    "label": "Current / history of trombembolism",
-                    "concept": "currentOrHistoryOfTrombembolism"
+                    "label": "Active liver disease / jaundice",
+                    "concept": "9a1003a7-27b0-42c7-804e-41f35bc5322e"
                   },
                   {
-                    "label": "Pregnancy",
-                    "concept": "pregnancy"
+                    "label": "BP > 140/90 and migraine with aura",
+                    "concept": "ccf9216a-ee7d-4f77-b6aa-b352f22b71b0"
                   },
                   {
-                    "label": "Diabetes for > 20 years",
-                    "concept": "diabetesForMoreThan20Years"
+                    "label": "BP > 160/100",
+                    "concept": "495831be-128a-4827-902c-06b7c76fdf31"
                   },
                   {
```

Comment:
Do the same for these answers. You could look at the order of changes or follow this column in the optionSets

<img width="700" alt="Screenshot 2025-03-20 at 15 22 34" src="https://github.com/user-attachments/assets/77bbf7cd-489f-41f7-806d-592701ce0e34" />